src/applications/personalization/profile/tests/util/index.unit.spec.js

```diff
@@ -62,176 +62,192 @@ describe('profile utils', () => {
       expect(eventDataObject).to.deep.equal(defaultDataObject);
     });
     it('returns the correct data when a bad address error is passed', () => {
-      const eventDataObject = createCNPDirectDepositAnalyticsDataObject([
-        {
-          title: 'Unprocessable Entity',
-          detail: 'One or more unprocessable user payment properties',
-          code: '126',
-          source: 'EVSS::PPIU::Service',
-          status: '422',
-          meta: {
-            messages: [
-              {
-                key: 'cnp.payment.generic.error.message',
-                severity: 'ERROR',
-                text:
-                  'Generic CnP payment update error. Update response: Update Failed: Required field not entered for mailing address update',
-              },
-            ],
+      const eventDataObject = createCNPDirectDepositAnalyticsDataObject({
+        errors: [
+          {
+            title: 'Unprocessable Entity',
+            detail: 'One or more unprocessable user payment properties',
+            code: '126',
+            source: 'EVSS::PPIU::Service',
+            status: '422',
+            meta: {
+              messages: [
+                {
+                  key: 'cnp.payment.generic.error.message',
+                  severity: 'ERROR',
+                  text:
+                    'Generic CnP payment update error. Update response: Update Failed: Required field not entered for mailing address update',
+                },
+              ],
+            },
           },
-        },
-      ]);
+        ],
+      });
```

Comment:
All the tests in this file are essentially the same, except I had to change the parameters passed into the `createCNPDirectDepositAnalyticsDataObject` for each test to be an object with an error key instead of just the error.